doc/z-chap05.xml

```diff
@@ -344,15 +344,16 @@
     &GAP; provides a way to define matrices which are in the
     filter <Ref Filt="IsMatrix" BookName="ref"/>.
     For technical reasons, the matrix semigroup functions in &Semigroups; rely
-    on a custom wrapper for matrices <Ref Filt="IsMatrixOverSemiring"/>.<P/> 
+    on a custom wrapper for matrices <Ref Filt="IsMatrixOverSemiring"/>.
     We take precautions to hide this fact from the user of &Semigroups; and
-    also provide conversion functions between the two representations.
+    also provide conversion functions between the two representations. <P/>
```

Comment:
I'm not sure where this paragraph came from, but I don't think it is accurate, and I think the whole paragraph should be deleted. I think this is a left over from the Semigroups 2.8 doc, written by @markuspf, which was true at the time, but is no longer the case.